components/istio-kyma-patch/README.md

```diff
@@ -0,0 +1,5 @@
+# Istio Kyma patch
+
+## Overview
+
+Application performs patch of already existing istio installation so kyma can be run.  
```

Comment:
We should describe here what changes are being made. 

For example: sidecar injector policy is changed to disabled, istio telemetry is pushing data to zipkin(jaeger) in the kyma-system etc.